bluepages/app/static/app/scss/bluepages.scss

```diff
@@ -105,17 +112,32 @@ main {
   margin-bottom: 3rem;
 }
 
+#results-column div.contact-results div.spinner-border {
```

Comment:
You could nest this in the `#results-column` styles above.

Reply:
Updated in following commit: ca6bf431d2dd5803aeafdf969b90ea5832b9dcc4